src/Frost.ts

```diff
@@ -312,6 +312,28 @@ export class Frost {
       })
   }
 
+  createApiToken(token: string): Promise<{ apiToken: string }> {
+    const options = {
+      method: Method.POST,
+      headers: new Headers({
+        'Content-Type': 'application/json',
+        token
+      })
+    }
+
+    const request = fetch(`${this.host}${Path.TOKENS}`, options)
+
+    return Promise.race([request, this.timeoutPromise()])
+      .then(async (value: any) => {
+        if (value.ok) return await value.json()
+
+        throw await value.text()
+      })
+      .catch(e => {
```

Comment:
What's the purpose of `catch`ing and then `throw`ing?

Reply:
What do you think?

Reply:
Yeah but if you're doing nothing with the error you caught, then why catch it?

Reply:
You're right, I don't need the catch. I will remove it. 👍